app/submitter/submitter.py

```diff
@@ -141,3 +141,36 @@ def send_message(self, message, tx_id, questionnaire_id, case_id=None):
         finally:
             if connection:
                 self._disconnect(connection)
+
+
+class GCSFeedback:
+    def __init__(self, bucket_name):
+        client = storage.Client()
+        self.bucket = client.get_bucket(bucket_name)
+
+    def upload(self, data, metadata):
+        blob = self.bucket.blob(metadata["object_key"])
+        blob.metadata = metadata
+        blob.upload_from_string(
+            str(data).encode("utf8"), content_type="application/json"
+        )
+
+        return True
+
+
+class LogFeedback:
+    @staticmethod
+    def upload(data, meta_data):
+        logger.info("uploading feedback")
+        logger.info(
+            "feedback payload",
+            data=data,
+            feedback_count=meta_data["feedback_count"],
+            form_type=meta_data["form_type"],
+            language_code=meta_data["language_code"],
+            object_key=meta_data["object_key"],
+            region_code=meta_data["region_code"],
+            tx_id=meta_data["tx_id"],
```

Comment:
This can be done with:
```suggestion
            **meta_data,
```

But as we're logging the data under a `data` key, I think this should be logged as `meta_data=meta_data`.